### DISCUSSION TITLE
Add possibility to modify emotes with BetterTTV and FrankerFaceZ effects

### DISCUSSION THREAD

**Discussion by lupa44** [Chatterino/chatterino2#5479] [Ideas]
Dear chatterino support Is there an option in the settings to include effects for emots? If not, could you add them? 
bttv effects are before the emote and ffz effects are after the emote, it looks like this 
 FFZ effect :
twitch chat https://gachi.gay/PcEZo
chatterino https://gachi.gay/atQE-
 Bttv effect :
twitch chat https://gachi.gay/zxiEX
chatterino https://gachi.gay/Wlsg4

**Comment by Felanbird** [comment]
Duplicate of #4470 & #4485
